__tests__/api.test.js

```diff
@@ -128,6 +128,37 @@ describe('GET requests', () => {
             expect(Object.keys(secondUser).length).toBe(3)
         })
     })
+    test('11 200: returns the correct article object with comment count when given an article_id ', () => {
+        return request (app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then(({body: {article}}) => {
+            const [firstArt] = article
+            expect(firstArt).toHaveProperty('comment_count', '11')
+            expect(firstArt).toHaveProperty('article_id', 1)
+            expect(Object.keys(firstArt).length).toBe(9)
```

Comment:
I dont know if you need this assertion, hopefully it was covered in previous tests, but its totes fine that its here